app/views/article/upload_form.html.slim

```diff
@@ -0,0 +1,16 @@
+=javascript_include_tag 'application'
+.litebox-embed(style="width:550px")
+  h1 Upload Subjects
+  p To create subjects from an external authority file, upload a CSV file with one row per subject.  (See #{link_to "example file", article_upload_example_path, {target: "_blank"}}.) 
```

Comment:
Instead of "See #{link_to "link", article_upload_example_path, {target: "_blank"}} for an example." Just link the "spreadsheet with one row per subject" words. That will mirror what we did for metadata.yml.